integrated_channels/blackboard/admin/__init__.py

```diff
@@ -19,8 +33,6 @@ class BlackboardEnterpriseCustomerConfigurationAdmin(admin.ModelAdmin):
     """
     list_display = (
         "enterprise_customer_name",
-        "client_id",
```

Comment:
should we leave these as optional, so we can use per-custom config when needed? it would also let us leaving the existing, working configs in place as-is - why risk disrupting working stuff, ya know?

Reply:
nevermind this is just the admin view.